src/histogram.rs

```diff
@@ -246,8 +247,14 @@ impl Histogram {
 }
 
 impl Histogram {
-    /// `observe` adds a single observation to the `Histogram`.
-    pub fn observe(&self, v: f64) {
+    /// `observe_seconds` adds a single observation (with seconds) to the `Histogram`.
+    pub fn observe_seconds(&self, v: f64) {
```

Comment:
observe_duration is the standard naming - but you need some form of timer first to handle all this for the user.